### PR TITLE
feat(#62): Persist namespace in dgraph

### LINKED ISSUE #62: Persist Namespace resource in dgraph
**Is this a BUG REPORT or FEATURE REQUEST?**:
 /kind feature

**What happened**:
Watch and persist namespace resource.

### PR DESCRIPTION
**Pull request by kreddyj**
**What this PR does / why we need it**:
1. Persist namespace in dgraph
1. Watch namespaces

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #62 

**Special notes for your reviewer**:
Tested on my local Kubernetes setup

### CODE REVIEW

File: pkg/controller/eventprocessor/processor.go [hunk 1]
```diff
@@ -98,6 +98,16 @@ func PersistPayloads(payloads []*interface{}) {
 			if err != nil {
 				log.Errorf("Error while persisting node %v", err)
 			}
+		} else if payload.ResourceType == "Namespace" {
```
Comment (gurusreekanthc) [inline pkg/controller/eventprocessor/processor.go]: we are not listening for namespace events, can you please introduce the same.

File: pkg/controller/dgraph/models/container.go [hunk 2]
```diff
@@ -51,6 +52,10 @@ func newContainer(containerXid, containerName, podUID string, pod api_v1.Pod) (*
 		StartTime:   pod.GetCreationTimestamp().Time,
 		Pod:         Pod{ID: dgraph.ID{UID: podUID, Xid: pod.Namespace + ":" + pod.Name}},
 	}
+	namespaceUID, err := createOrGetNamespaceByID(pod.Namespace)
```
[the remark this answers is not in the document]
Comment (kreddyj) [reply pkg/controller/dgraph/models/container.go]: ok

File: pkg/controller/eventprocessor/processor.go (hunk 1 above)
Comment (kreddyj) [reply pkg/controller/eventprocessor/processor.go]: ok

**Comment by kreddyj** [comment]
@gurusreekanthc made changes based on your comments. Will merge it after travis build passes

**Pull request merged**